cinema/serializers.py

```diff
@@ -78,3 +80,31 @@ class MovieSessionDetailSerializer(MovieSessionSerializer):
     class Meta:
         model = MovieSession
         fields = ("id", "show_time", "movie", "cinema_hall")
+
+
+class TicketSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ticket
+        fields = ("id", "movie_session","row", "seat")
+
+    def validate(self, attrs):
+        Ticket.validate_data(
+            attrs["seat"],
+            attrs["row"],
+            attrs["movie_session"].cinema_hall.seats_in_row,
+            attrs["movie_session"].cinema_hall.rows,
+            serializers.ValidationError
```

Comment:
The parameters passed to `Ticket.validate_data` are in the wrong order. The correct order should be `attrs['seat']`, `attrs['movie_session'].cinema_hall.seats_in_row`, `attrs['row']`, `attrs['movie_session'].cinema_hall.rows`, `serializers.ValidationError`.